_includes/blog/post_info.liquid

```diff
@@ -12,4 +12,7 @@
       {{ site.data.language.str_months[x] | default: date | date: "%B" }} {{ date | date: "%d, %Y" }}
     </p>
   {%- if author.url -%}</a>{%- endif -%}
+  {% if include.last_modified %}
+    <p class="meta">Last updated: {{ include.last_modified | date: site.data.language.str_date_format | default: '%B %-d, %Y' }}</p>
```

Comment:
I might add a special coass for it (to be smaller) or add it at the bottom as an edit a bit like in docusaurus.
But it's a good first step 👍